rules/no-new-enzyme-test/no-new-enzyme-test.js

```diff
@@ -0,0 +1,24 @@
+const { execSync } = require('child_process');
+
+module.exports = {
+  meta: {
+    type: 'problem'
+  },
+  create: context => {
+    // Get the newly added files and create a set of their names
+    const newFiles = new Set(execSync('git diff --cached --name-only --diff-filter=A').toString().trim().split('\n'));
```

Comment:
Would it be worth using [a library](https://www.npmjs.com/package/simple-git) to make this more robust?
```js
const git = require('simple-git')
...
const { files } = await git().diffSummary({ '--diff-filter': 'A', '--cached': true });
const fileNames = files.map(f => f.file);
```

Reply:
I thought about it. But then I thought that installing another library for a single rule will be too much. Especially because there is a native way to do it